apps/etterlatte-utbetaling/src/main/kotlin/utbetaling/config/ApplicationProperties.kt

```diff
@@ -38,7 +39,9 @@ data class ApplicationProperties(
                 serviceUserPassword = value("srvpwd"),
                 leaderElectorPath = value("ELECTOR_PATH"),
                 grensesnittavstemmingEnabled = value("GRENSESNITTAVSTEMMING_ENABLED").toBoolean(),
-                konsistensavstemmingEnabled = value("KONSISTENSAVSTEMMING_ENABLED").toBoolean()
+                konsistensavstemmingEnabled = value("KONSISTENSAVSTEMMING_ENABLED").toBoolean(),
```

Comment:
Tenker det gir mening å utvide dette litt med en pr type pr saktype, noe ala 
`BP_GRENSESNITTAVSTEMMING_ENABLED`, `BP_KONSISTENSAVSTEMMING_ENABLED` osv. Da kan vi da kan vi slå av og på begge typer avstemming for begge ytelser